ecommerce/Model/Image.php

```diff
@@ -10,4 +10,8 @@ class Image extends Model
         'product_id',
         'image'
     ];
+
+    public function products(){
```

Comment:
quan hệ trả về 1 bản ghi thì đặt tên số ít 
```suggestion
    public function product(){
```